README.md

```diff
@@ -9,97 +9,9 @@ For an overview and usage instructions, please visit [contributor-covenant.org](
 
 ## Contributing
 
-Please feel free to submit pull requests or open issues to improve the language of this pledge.
+Please feel free to submit pull requests or open issues to improve the language
+of this document or to provide additional [translations](http://contributor-covenant.org/version/1/3/0/i18n/).
```

Comment:
Should this point to version 1.4 if that is going to end up being the released version?